dev-go/gox/files/gox-1.0.1-add-riscv-support.patch

```diff
@@ -0,0 +1,550 @@
+--- a/go.go	2019-04-10 23:02:03.000000000 +0800
```

Comment:
Could you add more information about the patch (where it's taken from, author, upstream status)

Reply:
Yap, I'll add more information and force push